packages/paste-core/components/inline-code/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@twilio-paste/inline-code",
+  "version": "0.0.0",
+  "category": "typography",
+  "status": "production",
+  "description": "An inline code component visually distinguishes pieces of texts as fragments of code.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "peerDependencies": {
+    "@twilio-paste/box": "^7.0.1",
```

Comment:
chore: make sure you're capturing the full dependency tree for peer deps. https://github.com/twilio-labs/paste/blob/3d04f3d9afe509d19d2e7ce7ce10470557bede81/README.md#peer-deps

Reply:
Ah thank you!